examples/nlp/language_modeling/prepare_truncated_data.py

```diff
@@ -24,7 +27,9 @@
 """
 
 
-def _process_line(line, task, tokenizer, max_seq_length, tokens_to_generate_extra, prompt, remove_newline_tab=False):
+def _process_line(
+    line, task, tokenizer, max_seq_length, tokens_to_generate_extra, prompt, truncation_pos, remove_newline_tab=False
```

Comment:
Can we add a default value to the `truncation_pos` parameter like:  
`truncation_pos='right'`

Reply:
Yes, this is just to keep things consistent everywhere. The function definition could be:
```
def _process_line(
    line,
    task,
    tokenizer, max_seq_length,
    tokens_to_generate_extra,
    prompt,
    truncation_pos='right', 
    remove_newline_tab=False
):
```